docs/docs/quickstart-inference.md

```diff
@@ -0,0 +1,295 @@
+
+In this guide, you'll:
+- Set up Weave and W&B Inference
+- Build a basic LLM application with automatic tracing
+- Compare multiple models
+- Evaluate model performance on a dataset
+- View your results in the Weave UI
+
+## Prerequisites
+
+Before you begin, you need a [W&B account](https://app.wandb.ai/login?signup=true) and an API key from from [https://wandb.ai/authorize](https://wandb.ai/authorize).
+
+Then, in a Python environment running version 3.9 or later, install the required libraries: 
+
+```bash
+pip install weave openai
+```
+
+The `openai` library is installed because you use the standard `openai` client to interact with W&B Inference, regardless of which hosted model you're actually calling. This allows you to swap between supported models by only changing the slug, and make use of any existing code you have that was written to use the OpenAI API. 
+
+## Step 1: Trace your first LLM call
+
+Start with a basic example that uses Llama 3.1-8B through W&B Inference. 
+
+When you run this code, Weave:
+- Traces your LLM call automatically
+- Logs inputs, outputs, latency, and token usage
+- Provides a link to view your trace in the Weave UI
+
+```python
+import weave
+import openai
+
+# Initialize Weave - replace with your-team/your-project
+weave.init("my-first-weave-project")
+
+# Create an OpenAI-compatible client pointing to W&B Inference
+client = openai.OpenAI(
+    base_url='https://api.inference.wandb.ai/v1',
+    api_key="YOUR_WANDB_API_KEY",  # Replace with your actual API key
+    project="my-first-weave-project",  # Required for usage tracking
+)
+
+# Decorate your function to enable tracing; use the standard OpenAI client
+@weave.op()
+def ask_llama(question: str) -> str:
+    response = client.chat.completions.create(
+        model="meta-llama/Llama-3.1-8B-Instruct",
+        messages=[
+            {"role": "system", "content": "You are a helpful assistant."},
+            {"role": "user", "content": question}
+        ],
+    )
+    return response.choices[0].message.content
+
+# Call your function - Weave automatically traces everything
+result = ask_llama("What are the benefits of using W&B Weave for LLM development?")
+print(result)
+```
+
+## Step 2: Build a text summarization application
+
+Next, try running this code, which is a basic summarization app that shows how Weave traces nested operations:
+
+```python
+import weave
+import openai
+
+# Initialize Weave - replace with your-team/your-project
+weave.init("my-first-weave-project")
+
+@weave.op()
+def extract_key_points(text: str) -> list[str]:
+    """Extract key points from a text."""
+    response = client.chat.completions.create(
+        model="meta-llama/Llama-3.1-8B-Instruct",
+        messages=[
+            {"role": "system", "content": "Extract 3-5 key points from the text. Return each point on a new line."},
+            {"role": "user", "content": text}
+        ],
+    )
+    return response.choices[0].message.content.strip().split('\n')
```

Comment:
`splitlines()` is preferable to `split('\n')`. Also, at the risk of complicating the code we might want to filter out the blank lines so things look nicer in Weave?

<img width="996" height="354" alt="Image" src="https://github.com/user-attachments/assets/f879568e-44d6-4e26-9804-7eaf21bb2c10" />